deploy/servctl_utils/deploy_command_utils.py

```diff
@@ -200,22 +200,6 @@ def deploy_redis(settings):
 def deploy_seqr(settings):
     print_separator("seqr")
 
-    if settings["BUILD_DOCKER_IMAGES"]:
```

Comment:
`settings["BUILD_DOCKER_IMAGES"]` is False by default so maybe leave the `if` check here and throw an explicit error if someone tries to run this command to build the image?

Reply:
ok 👍